packages/core/frontend/sirius-components-core/src/materialui.ts

```diff
@@ -18,6 +18,21 @@ declare module '@material-ui/core/styles' {
       leftBackground: React.CSSProperties['color'];
       rightBackground: React.CSSProperties['color'];
     };
+    navigationBar: {
+      border: React.CSSProperties['color'];
+      background: React.CSSProperties['color'];
+    };
+    diagram: {
+      selectedNode: React.CSSProperties['color'];
+      selectedEdge: React.CSSProperties['color'];
+    };
```

Comment:
The component `sirius-components-core` cannot possibly be aware of the existence of a `diagram` representation.

So either you only need to customize the `selected` color which could be done here, or you really want to customize diagram specific colors, which I don't mind, but such code will have to be in `sirius-components-diagrams` (with a dedicated `materialui.ts`). In this case, the fallback theme should probably move from `sirius-components-core` to `sirius-web-application` since it would not be possible to add diagram-specific colors in `sirius-components-core`.

Reply:
Done, I oped for the `selected` variable choice instead of moving fallback theme into sirius-web-application.